signup-service/src/main/java/io/mosip/signup/services/IdentityVerificationService.java

```diff
@@ -178,4 +206,19 @@ private IdentityVerifierDetail[] getIdentityVerifierDetailsFromConfigServer() {
         }
         return verifierDetails;
     }
+
+    private PrivateKey loadPrivateKey(String alias, String cyptoPassword) {
+        try {
+            File directory = ResourceUtils.getFile("classpath:"+p12FileName);
+            FileInputStream fis = new FileInputStream(directory);
+            KeyStore ks = KeyStore.getInstance("PKCS12");
+            ks.load(fis, cyptoPassword.toCharArray());
+
+            // Retrieve the private key
+            return  (PrivateKey) ks.getKey(alias, cyptoPassword.toCharArray());
+        }catch (Exception e) {
+            log.error("Failed to load private key from keystore", e);
+            throw new SignUpException("private_key_load_failed");
```

Comment:
Let's create a constant for this error code.

Reply:
done